src/components/10-atoms/carousel/README.md

```diff
@@ -1,21 +1,34 @@
 # Carousel
 
-Every child is rendered as a slide. Only one child is visible at one time.
+Each child is rendered as a slide. In default mode, only one child is visible at a time, and navigation arrows
+are provided. Also, auto-rotation to the right is enabled by default.
 
 ## Properties
 
 ### autorotatedisabled
 
-This Boolean attribute specifies if the slides will not automatically rotate.
+This Boolean attribute specifies whether slide auto-rotation should be suppressed (default: false).
 
 ### autorotatetime
 
-This number specifies the milliseconds the slides will automatically rotate.
+This numeric attribute specifies the waiting time in milliseconds for the next slide to appear in auto-rotation mode (default: 5000).
 
 ### keysenabled
 
-This boolean attribute specifies if the keyboard navigation (left/right keys) should be enabled.
+This Boolean attribute specifies whether keyboard navigation via left/right arrow keys should be enabled (default: false).
+
+### arrowstyle
+
+This string-valued attribute adds custom inline styling to the arrow buttons of the carousel
+(e.g. `border-color:grey` for grey arrows, or `display:none` to suppress arrows).
```

Comment:
The user can do anything with css now. Isnt it too risky that we get carousels that have "crazy" arrows?

Reply:
I also don't like that we hand down CSS properties as HTML properties. This is a bit sandboxy and allows for too much variants. Button, Button-Link, Link, Hero Banner, ... almost everywhere we have a set of colors that we allow.

Reply:
Our users will like it _because_ it gives them flexibility. This flexibility is indeed necessary, to both allow backwards compatibility and new uses, see the mocked AXA main page in the demos.

To provide this with only one tiny change is arguably elegant, so I would defend it here.